### ISSUE TITLE
New Fc multi position

### ISSUE THREAD

**Issue by bbday** [Regista6/FIFA-23-Automated-SBC-Solving#5]
Do you have plan to fix it for new fc24? 
cause now is possible same player can autoswitch position then a CM player can be also at same time RB and get bonus for chemistry

**Comment by Regista6** [comment]
Yeah, I've asked the author of the tool I am using to download Squad CSV to include an additional column for `Alternate Positions`. Once that is available, this can be implemented. Apart from this, I think only the chemistry of Icons has changed; they now add 1 chem to every league!

**Comment by bbday** [comment]
if you need csv that include alternative position no issue i can generate it, usually each player have 2-3 alternative position (let me know it format that you wanna it, like array [CM,LM,RM] example) 
Let me know it

**Comment by bbday** [comment]
You can use this CSV to make test i use all fifa code then more accurate

https://drive.google.com/file/d/10HFvMCv1-Nw6T7hohPctHO_RbIWaUtWG/view?usp=drive_link

**Comment by Regista6** [comment]
The tool has been updated. Currently, the dataset looks like this:

| Name    | Rating | Rarity | Preferred Position | Nation     | League       | Team   | Price Limits | Last Sale Price | Discard Value | Contract | Untradeable | Loans | DefinitionId | IsDuplicate | IsInActive11 | Alternate Positions | ExternalPrice |
|---------|--------|--------|-------------------|------------|--------------|--------|--------------|-----------------|---------------|----------|-------------|-------|--------------|-------------|--------------|----------------------|---------------|
| Miedema | 86     | Rare   | ST                | Netherlands | Barclays WSL | Arsenal| --NA--       | 0               | 0             | 7        | TRUE        | TRUE  | 233746       | FALSE       | TRUE         | CAM,CF,ST            | 7000          |


I am thinking of just creating separate entries of a particular player for each position. Also would like to access your dataset as well. Thanks!

**Comment by bbday** [comment]
my data comming from EA player iteration.
Atm main change is same player have mutiple positions and chemistry bonus for club is valid if club name is name (male/female) cause clubId is different number.

I think to use extra value like ClubMFId use same code id male/female for chemistry calc
For multiple position atm i dont have any idea except your but will increase 4 iterations

**Comment by Regista6** [comment]
Many thanks for taking an interest in this project. Can you grant access to your drive link above? 

In the above example, Miedema has club listed as Arsenal. If for female players, this consistency is maintained, then it will be fine, I think! 

Dataset size will definitely increase (x3 approximately if we consider each player can play in 3 positions). But how much it will impact solver performance remains to be seen.

**Comment by bbday** [comment]
sorry i didnt realize need access to see it

**Comment by Regista6** [comment]
In your dataset, male and female players have different club IDs for the same club name. One of my intentions was to combine players from club and market. But for that, I would need the entire dataset of players with prices. For now, the extension to download club csv works well. Let's see! 

https://github.com/Regista6/EA-FC-24-Automated-SBC-Solving/commit/a8f3ea4cd804f8858fe407d9e7d25ac0c5d30e84
